tests/cases/fourslash/codeFixAddConvertToUnknownForNonOverlappingTypes1.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference path='fourslash.ts' />
+
+////[|0 as string|]

```

Comment:
Just use `newFileContent` since the entire file is the range anyway -- then no need for `[||]`